static/app/views/issueList/actions/index.tsx

```diff
@@ -331,7 +331,7 @@ class IssueListActions extends React.Component<Props, State> {
 
 const Sticky = styled('div')`
   position: sticky;
-  z-index: ${p => p.theme.zIndex.header};
+  z-index: 1;
```

Comment:
Can we keep these in `theme` so that they're a little bit more self documenting? Otherwise this could end up with a bunch of ever-increasing z-index battles. If they were in theme, we would have a central location to see "all" relevant zIndex values so you have a sense of what you're affecting when adding/changing z-index values. I think it'd be fine to have these specific for the issues list

Reply:
i haven't thought of a way to do that, right now they just have to be one bigger than whatever component is below it. Adding another dropdown would change them around again

Reply:
could use initial here

Reply:
yeah i think this can just be theme.zIndex.issuesList.whateverDropdown, theme.zIndex.issuesList.fooDropdown, theme.zIndex.issuesList.barDropdown, etc